SmartDeviceLinkTests/DevAPISpecs/SDLTextAndGraphicManagerSpec.m

```diff
@@ -491,9 +491,9 @@ @interface SDLTextAndGraphicUpdateOperation ()
 
             testWindowCapability = [[SDLWindowCapability alloc] initWithWindowID:@(SDLPredefinedWindowsDefaultWindow) textFields:nil imageFields:nil imageTypeSupported:nil templatesAvailable:nil numCustomPresetsAvailable:nil buttonCapabilities:nil softButtonCapabilities:nil menuLayoutsAvailable:nil dynamicUpdateCapabilities:nil];
             testDisplayCapability = [[SDLDisplayCapability alloc] initWithDisplayName:@"Test display" windowCapabilities:@[testWindowCapability] windowTypeSupported:nil];
-            testSystemCapability = [[SDLSystemCapability alloc] initWithDisplayCapabilities:@[testDisplayCapability]];
 
-            [testManager sdl_displayCapabilityDidUpdate:testSystemCapability];
+            OCMExpect([mockSystemCapabilityManager defaultMainWindowCapability]).andReturn(testWindowCapability);
+            [testManager sdl_displayCapabilityDidUpdate];
```

Comment:
Verify?

Reply:
Must have been a copy and paste error. Switched to `OCMStub`